### PR TITLE
added helper method for future refactoring

### PR DESCRIPTION
**Pull request by heejaechang**

### CODE REVIEW

File: packages/pyright-internal/src/languageService/indentationUtils.ts [hunk 1]
```diff
@@ -421,26 +418,364 @@ function _getLineEndingLength(parseResults: ParseResults, line: number) {
     return length;
 }
 
-function _getIndentationFromText(parseResults: ParseResults, line: number) {
+function _getIndentationFromText(
```
Comment (erictraut) [inline packages/pyright-internal/src/languageService/indentationUtils.ts:421]: For consistency with the rest of the code base, can you move private functions below exported functions within the module?

File: packages/pyright-internal/src/languageService/indentationUtils.ts [hunk 2]
```diff
@@ -421,26 +418,364 @@ function _getLineEndingLength(parseResults: ParseResults, line: number) {
...
+            texts.push(info.text);
+        }
+
+        previousInfo = info;
+    }
+
+    return texts.join('');
+}
+
+interface TokenInfo extends TextRange {
```
Comment (erictraut) [inline packages/pyright-internal/src/languageService/indentationUtils.ts:535]: For consistency with the rest of the code base, please move interface definitions before functions in the file.

File: packages/pyright-internal/src/parser/tokenizer.ts [hunk 3]
```diff
@@ -135,6 +135,8 @@ const _byteOrderMarker = 0xfeff;
 
 const _maxStringTokenLength = 32 * 1024;
 
+export const DefaultTabSize = 8;
```
Comment (erictraut) [inline packages/pyright-internal/src/parser/tokenizer.ts:138]: For consistency with the rest of the code base, please make constant names lower case.

**Review by bschnurr — APPROVED** [review]

**Comment by bschnurr** [comment]
PR name seems wrong

File: packages/pyright-internal/src/languageService/indentationUtils.ts [hunk 4]
```diff
@@ -30,6 +32,8 @@ import {
     TokenType,
 } from '../parser/tokenizerTypes';
 
+export const TabSize = 8;
```
Comment (erictraut) [inline packages/pyright-internal/src/languageService/indentationUtils.ts:35]: For consistency with the rest of the code base, please make constant names lowercase.

**Review by erictraut — COMMENTED** [review]
Overall, looks pretty good. Just a couple of style consistency suggestions.

File: packages/pyright-internal/src/languageService/indentationUtils.ts (hunk 1 above)
Comment (heejaechang) [reply packages/pyright-internal/src/languageService/indentationUtils.ts:421]: sure.

File: packages/pyright-internal/src/languageService/indentationUtils.ts (hunk 2 above)
Comment (heejaechang) [reply packages/pyright-internal/src/languageService/indentationUtils.ts:535]: okay

File: packages/pyright-internal/src/parser/tokenizer.ts (hunk 3 above)
Comment (heejaechang) [reply packages/pyright-internal/src/parser/tokenizer.ts:138]: okay

File: packages/pyright-internal/src/languageService/indentationUtils.ts (hunk 4 above)
Comment (heejaechang) [reply packages/pyright-internal/src/languageService/indentationUtils.ts:35]: okay

**Comment by heejaechang** [comment]
@erictraut addressed your feedbacks.

**Pull request merged**